controls/cis_eks.yml

```diff
@@ -0,0 +1,647 @@
+policy: CIS
+title: Configuration Recommendations of an EKS System
+id: cis_eks
+version: '1.0.1'
+source: https://www.cisecurity.org/cis-benchmarks/
+levels:
+- id: level_1
+- id: level_2
+  inherits_from:
+  - level_1
+
+controls:
+- id: '1'
+  title: '1 Control Plane Components'
+  description: |-
+    Security is a shared responsibility between AWS and the Amazon EKS customer. The shared responsibility model describes this as security of the cloud and security in the cloud:
+
+    Security of the cloud – AWS is responsible for protecting the infrastructure that runs AWS services in the AWS Cloud. For Amazon EKS, AWS is responsible for the Kubernetes control plane, which includes the control plane nodes and etcd database. Third-party auditors regularly test and verify the effectiveness of our security as part of the AWS compliance programs. To learn about the compliance programs that apply to Amazon EKS, see AWS Services in Scope by Compliance Program.
+
+    Security in the cloud – Your responsibility includes the following areas.
+
+        - The security configuration of the data plane, including the configuration of the security groups that allow traffic to pass from the Amazon EKS control plane into the customer VPC
+        - The configuration of the worker nodes and the containers themselves
+
+          - The worker node guest operating system (including updates and security patches)
+              - Amazon EKS follows the shared responsibility model for CVEs and security patches on managed node groups. Because managed nodes run the Amazon
+                EKS-optimized AMIs, Amazon EKS is responsible for building patched versions of these AMIs when bugs or issues are reported and we are able to publish a fix. However, customers are responsible for deploying these patched AMI versions to your managed node groups.
+
+        - Other associated application software:
+          - Setting up and managing network controls, such as firewall rules
+          - Managing platform-level identity and access management, either with or in addition to IAM
+
+        - The sensitivity of your data, your company’s requirements, and applicable laws and regulations
+
+    AWS is responsible for securing the control plane, though you might be able to configure certain options based on your requirements.
+  notes: |-
+    As noted, evaluating the security of the control plane is not the
+    responsibility of customers or deployers, but of Amazon instead. Thus, this
+    profile won't cover this.
```

Comment:
nit: s/won't cover this/will not implement controls from Section 1/ 

Or something of that nature?